<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Searchdetails</class>
 <widget class="QDialog" name="Searchdetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Searchdetails
{
	background:#2D2D2D;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>3</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)
</string>
   </property>
   <property name="text">
    <string>                                     SEARCH  EMPLOYEE</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SearchBox">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>70</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>Employee ID</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Department</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Sname">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchBut">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#SearchBut
{
	background:#00855C;
	border:none;
	color:white;
}

#SearchBut:hover
{
	background:#2D2D2D;
	border:2px solid #00855C;
	color:#00855C;
}</string>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QTableView" name="EmptableView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>601</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="HomeButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgb(255, 176, 107);</string>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
  <widget class="QLabel" name="searchconnection">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>489</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
